packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1904,6 +1954,11 @@ export default function buildKernel(
           case 'crank-failed':
             policyOutput = policy.crankFailed(policyInput[1]);
             break;
+          case 'cleanup': {
+            const { didCleanup = () => true } = policy;
+            policyOutput = didCleanup(policyInput[1]);
```

Comment:
```suggestion
            // Give the policy a chance to interrupt kernel execution, but default to continuing.
            const { didCleanup } = policy;
            policyOutput = didCleanup ? didCleanup(policyInput[1]) : true;
```
It also occurs to me that the policy method names should have better emphasized their responsive nature (e.g., `onCrankComplete`, `onEmptyCrank`, `onCleanup`), but it might be weird for cleanup to be the odd one out.

Reply:
Yeah, I'm not satisifed with our policy language, and this new `didCleanup` makes me feel worse about it. I need to find a way to let the policy specify separate budgets for different phases of cleanup (deleting import and export c-list entries can be expensive, I want to limit those to 5-per-run, but cleaning up e.g. vatstore entries is cheap, we can afford to do 50 per run). So I'll be revisiting this before we ship, and I hope to use that opportunity to rewrite the policy API, modulo backwards-compatibility questions.